plugins/profefe.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: profefe
+spec:
+  version: v0.9.0
+  homepage: https://github.com/profefe/kube-profefe
+  shortDescription: Gather and manage pprof profiles from running pods
+  description: |
+    This plugin helps you do manage pprof profiles via Profefe. It helps you do
```

Comment:
There are some `do`s here that don't really make sense. Also, it helps readability if you use an active tense, like:

> This plugin helps to manage pprof profiles via Profefe. It allows to
collect profiles from running pods with Profefe and retrieve or inspect
profiles stored in Profefe.